x-pack/plugins/apm/public/components/shared/backend_link.tsx

```diff
@@ -13,8 +13,19 @@ import { useApmRouter } from '../../hooks/use_apm_router';
 import { truncate } from '../../utils/style';
 import { ApmRoutes } from '../routing/apm_route_config';
 import { SpanIcon } from './span_icon';
+import { TruncateWithoutTooltip } from './truncate_with_tooltip';
```

Comment:
the component name and the filename is a little bit confusing. 

Can you please separate the components or give a more insightful name

Reply:
This is done. Please check

Reply:
thanks for updating it